checker/tests/resourceleak/MustCallAliasPassthroughWrong3.java

```diff
@@ -6,6 +6,7 @@
 
 class MustCallAliasPassthroughWrong3 {
 
+  // :: warning: (mustcallalias.method.return.and.param)
```

Comment:
Should this warning be added, _or_ should the return type be annotated with `@MustCallAlias`?

Reply:
This test case was intended to cover the case when a user forgets an `@MustCallAlias` annotation on an otherwise correct implementation of a wrapper type. So, I think adding the warning is the right choice.